src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCarRental = 40;
+  const smallDiscount = 20;
+  const BigDiscount = 50;
+
+  let amount = days * dailyCarRental;
+
+  if (days >= 7) {
+    amount -= BigDiscount;
+  } else if (days >= 3) {
+    amount -= smallDiscount;
+  } else {}
```

Comment:
```suggestion
  }
```
why did you add this?

Reply:
I thought that it must be put. What after (else if) must be (else)